### PR TITLE
mkdocs-material: 9.3.1 -> 9.4.14

### PR DESCRIPTION
**Pull request by caarlos0**
## Description of changes

Updates mkdocs-material to latest

Needs #270307 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/mkdocs-material/default.nix [hunk 1]
```diff
@@ -12,14 +12,15 @@
...
 , pythonOlder
 , regex
 , requests
 }:
 
 buildPythonPackage rec {
   pname = "mkdocs-material";
-  version = "9.3.1";
+  version = "9.4.14";
   format = "pyproject";
```
Comment (dotlambda) [inline pkgs/development/python-modules/mkdocs-material/default.nix:24]:
```suggestion
  pyproject = true;
```

File: pkgs/development/python-modules/mkdocs-material/default.nix [hunk 2]
```diff
@@ -60,6 +62,6 @@ buildPythonPackage rec {
     description = "Material for mkdocs";
```
Comment (dotlambda) [inline pkgs/development/python-modules/mkdocs-material/default.nix:62]:
```suggestion
    changelog = "https://github.com/squidfunk/mkdocs-material/blob/${src.rev}/CHANGELOG";
    description = "Material for mkdocs";
```

File: pkgs/development/python-modules/mkdocs-material/default.nix [hunk 3]
```diff
@@ -47,6 +48,7 @@ buildPythonPackage rec {
     pymdown-extensions
     regex
     requests
+    trove-classifiers
```
Comment (dotlambda) [inline pkgs/development/python-modules/mkdocs-material/default.nix:51]:
```suggestion
```
This should be in `nativeBuildInputs`.

**Comment by caarlos0** [comment]
thanks for the review @dotlambda 🙏

**Comment by dotlambda** [comment]
Can you rebase your branch on https://github.com/NixOS/nixpkgs/commit/f25aaf9b44b153b805154cababc6807d42a8a3fa?

**Comment by piegamesde** [comment]
It looks like you accidentally mass-pinged a bunch of people, which are now subscribed
and getting notifications for everything in this pull request. Unfortunately, they
cannot be automatically unsubscribed from the issue (removing review request does not
unsubscribe), therefore development cannot continue in this pull request anymore.

Please open a new pull request with your changes, link back to this one and ping the
people actually involved in here over there.

In order to avoid this in the future, there are instructions for how to properly
rebase between branches in our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging).
Setting your pull request to draft prior to rebasing is strongly recommended.
In draft status, you can preview the list of people that are about to be requested
for review, which allows you to sidestep this issue.
This is not a bulletproof method though, as OfBorg still does review requests even on draft PRs.

**Pull request closed without merging**